test/unit/double_free.c

```diff
@@ -101,10 +105,44 @@ TEST_BEGIN(test_small_double_free_tcache) {
 }
 TEST_END
 
+TEST_BEGIN(test_small_double_free_arena) {
+	test_skip_if(!config_debug);
+
+	test_skip_if(opt_debug_double_free_max_scan == 0);
+
+	bool tcache_enabled;
+	enable_tcache(&tcache_enabled);
+	test_skip_if(!tcache_enabled);
+
+	test_double_free_pre();
+	/* Allocate one more pointer so that the cache won't be empty after
+	 * flushing.
+	*/
```

Comment:
nit: Format is off. For comments that's only one word over a line, I'd say try trimming it down to fit in one line.

Reply:
trimmed and formatted